jupyter_server_ydoc/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 from .app import YDocExtension
 
-__version__ = "0.6.1"
+__version__ = "0.8.0"
```

Comment:
Isn't it the role of the releaser to bump the version?

Reply:
You can give the releaser the branch and specify the version (instead of `next`), so I would not change the version here.